packages/core/src/AbstractRenderer.ts

```diff
@@ -16,8 +45,25 @@ const tempMatrix = new Matrix();
  * @extends PIXI.utils.EventEmitter
  * @memberof PIXI
  */
-export class AbstractRenderer extends EventEmitter
+export abstract class AbstractRenderer extends EventEmitter
 {
+    public resolution: number;
+    public clearBeforeRender?: boolean;
+    public readonly options: IRendererOptions;
+    public readonly type: RENDERER_TYPE;
+    public readonly screen: Rectangle;
+    public readonly view: HTMLCanvasElement;
+    public readonly plugins: IRendererPlugins;
+    public readonly transparent: boolean | 'notMultiplied';
+    public readonly autoDensity: boolean;
+    public readonly preserveDrawingBuffer: boolean;
+
+    protected readonly _tempDisplayObjectParent: DisplayObject;
+    protected _backgroundColor: number;
+    protected _backgroundColorString: string;
+    _backgroundColorRgba: number[];
```

Comment:
In a different file you declare `private` explicitly, shouldn't that be followed here?
Also, it'd probably be a good idea to enforce explicit access modifiers in eslint if that's the way to go.

Reply:
That particular field is used in RenderTextureSystem. I've decided not to add modifiers that can affect code readability.